internal/controller/constants.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+const (
+	finalizerOrGateName string = "org.instaslice/accelerator"
```

Comment:
Just in case we'll need to distinguish them at some point:
```
gateName string = "org.instaslice/accelerator"
finalizerName string = gateName
```

Reply:
If we do we can make that change.

I lean towards YANGI for now.

Reply:
wdyt @asm582?

Reply:
Created https://github.com/openshift/instaslice-operator/issues/110

Reply:
I agree with the feedback, I did not put too much effort in naming as we are pending agreement on InstaSlice resource names, we should get back to that issue and the changes requested are orthogonal to this PR